register_forte_options.py

```diff
@@ -421,13 +424,13 @@ def register_aci_options(options):
 
     options.add_int("SCI_MAX_CYCLE", 20, "Maximum number of cycles")
 
-    options.add_bool("ACI_QUIET_MODE", False,
+    options.add_bool("SCI_QUIET_MODE", False,
```

Comment:
Should these moved to the sci options block?

Reply:
Thanks for catching this.